Src/Web/Web.Shared.Net/ClientIpHeaderTelemetryInitializer.cs

```diff
@@ -91,7 +91,7 @@ protected override void OnInitializeTelemetry(HttpContext platformContext, Reque
         private static string CutPort(string address)
         {
             // For Web sites in Azure header contains ip address with port e.g. 50.47.87.223:54464
-            int portSeparatorIndex = address.IndexOf(":", StringComparison.OrdinalIgnoreCase);
+            int portSeparatorIndex = address.LastIndexOf(":", StringComparison.OrdinalIgnoreCase);
```

Comment:
Per Paul's comments below, why not just use the IPAddress class here instead of doing raw string manipulation?